packages/styles/scss/components/__tests__/data-table-test.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright IBM Corp. 2018, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @jest-environment node
+ */
+
+'use strict';
+
+const { SassRenderer } = require('@carbon/test-utils/scss');
+
+const { render } = SassRenderer.create(__dirname);
+
+describe('scss/components/data-table', () => {
+  test('Public API', async () => {
+    const { unwrap } = await render(`
+       @use 'sass:map';
+       @use 'sass:meta';
+       @use '../data-table';
+ 
+       $_: get('mixin', meta.mixin-exists('data-table-v2-action', 'data-table-v2-action'));
+       $_: get('variables', map.keys(meta.module-variables('data-table-v2-action')));
+
+       $_: get('mixin', meta.mixin-exists('data-table-core', 'data-table-core'));
+       $_: get('variables', map.keys(meta.module-variables('data-table-core')));
+
+       $_: get('mixin', meta.mixin-exists('data-table-expandable', 'data-table-expandable'));
+       $_: get('variables', map.keys(meta.module-variables('data-table-expandable')));
+
+       $_: get('mixin', meta.mixin-exists('data-table-sort', 'data-table-sort'));
+       $_: get('variables', map.keys(meta.module-variables('data-table-sort')));
+     `);
+    expect(unwrap('mixin')).toBe(true);
```

Comment:
I think we might need to have a couple of tests for data table, one for each file we'll want teams to be able to import.

For the public API test, I think we'll only need two calls: one for mixins one for variables. The `meta.module-variables` should take in `data-table` as the argument since that's the namespace being loaded in.

We should also have similar style tests for each file we're expecting people to be able to bring in together 👀